dotnet/src/SemanticKernel.Core/Planning/Plan.cs

```diff
@@ -60,34 +60,16 @@ public sealed class Plan : ISKFunction
     [JsonPropertyName("next_step_index")]
     public int NextStepIndex { get; private set; }
 
-    #region ISKFunction implementation
-
-    /// <inheritdoc/>
-    [JsonPropertyName("name")]
-    public string Name { get; set; } = string.Empty;
-
     /// <inheritdoc/>
     [JsonPropertyName("plugin_name")]
     public string PluginName { get; set; } = string.Empty;
```

Comment:
Unrelated to this PR, what's the purpose of Plan.PluginName? We removed PluginName from KernelFunction... does it still make sense on Plan?

Reply:
I don't know but I'll add this to an issue to track followup on this PR